src/main/java/org/geysermc/discordbot/commands/moderation/ForumCommand.java

```diff
@@ -0,0 +1,301 @@
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/GeyserDiscordBot
+ */
+
+package org.geysermc.discordbot.commands.moderation;
+
+import com.jagrosh.jdautilities.command.*;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.channel.concrete.ForumChannel;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import net.dv8tion.jda.api.entities.channel.forums.ForumTag;
+import net.dv8tion.jda.api.entities.channel.forums.ForumTagSnowflake;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.managers.channel.concrete.ThreadChannelManager;
+import net.dv8tion.jda.api.utils.messages.MessageCreateData;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class ForumCommand extends SlashCommand {
+    public static final String FORUM_CHANNEL_ID = "1038995448100306964";
+    public ForumCommand() {
+        this.name = "post";
+        this.hidden = true;
+        this.help = "help tool to manage forum posts.";
+        this.guildOnly = true;
+        this.children = new SlashCommand[] {
+                new createPostSubCommand(),
+                new CloseOldPostSubCommand(),
+                new ClosePostSubCommand(),
+                new renamePostSubCommand(),
+                new addTagPostCommand(),
+                new removeTagPostCommand()
+        };
+    }
+
+    @Override
+    protected void execute(SlashCommandEvent event) {
+        // unused
+    }
+
+    public static class createPostSubCommand extends SlashCommand {
+        public createPostSubCommand() {
+            this.name = "create";
+            this.help = "create post";
+            this.userPermissions = new Permission[] {
+                    Permission.CREATE_PUBLIC_THREADS
+            };
+            this.options = Arrays.asList(
+                    new OptionData(OptionType.STRING, "title", "add the post title", true),
+                    new OptionData(OptionType.STRING, "issue", "add the post message/issue", true),
+                    new OptionData(OptionType.USER, "member", "The member to ping in the post", false)
+            );
+        }
+
+        @Override
+        protected void execute(@NotNull SlashCommandEvent event) {
+            String title = Objects.requireNonNull(event.getOption("title")).getAsString();
+            String targetUser = event.getOption("member") != null ? Objects.requireNonNull(event.getOption("member")).getAsUser().getAsMention() : "";
+            String issue = Objects.requireNonNull(event.getOption("issue")).getAsString();
+            if (!targetUser.isEmpty()) {
+                issue += " " + targetUser;
+            }
+            Guild guild = event.getGuild();
```

Comment:
```suggestion
            Guild guild = Checks.notNull(event.getGuild(), "Server");
```
JDA-Chewtils does this check internally, so this isn't an issue since we specified guildOnly.
Also don't need the check below